pages/findproduct.tsx

```diff
@@ -0,0 +1,219 @@
+import Main from '../layout/Main';
+
+export default function dashboard() {
+  const IntakeType: Array<string> = [
+    'tablet',
+    'Capsule',
+    'Injection',
+    'Powder',
+    'Syrup',
+    'Cream',
+    'oil',
+  ];
+  const Age: Array<string> = ['All', 'Child', 'Adult', 'Elderly'];
+  const Priscription: Array<string> = ['yes', 'no'];
+  const Brand: Array<string> = ['SBL', 'ADEL', 'Deve Herbs'];
+  return (
+    <Main>
```

Comment:
I looked into the deployment. I think this PR doesn't satisfies the project need.

Your're wrapping the whole page with the ```main``` widget which is not a good idea. 

![image](https://user-images.githubusercontent.com/63960580/160439973-58eec80f-6da3-4a27-afbe-f4732f44640e.png)

You may remove it and make this UI more clean and asthetic... I think the left side bar for sorting the medicine is not needed.